package/MDAnalysis/analysis/density.py

```diff
@@ -80,6 +80,10 @@
    :members:
    :inherited-members:
 
+.. versionchanged:: 0.15.0
+    The "permissive" flag is not used anymore (and effectively
+    defaults to True); it will be completely removed in 0.16.0.
```

Comment:
please can you say that it is deprecated. That way we can grep for  `deprecated` before a release to know what to remove.